core/src/main/scala/com/nvidia/spark/rapids/tool/qualification/QualOutputWriter.scala

```diff
@@ -1135,6 +1138,7 @@ object QualOutputWriter {
       appInfo.unSupportedExecs,
       appInfo.unSupportedExprs,
       appInfo.allClusterTagsMap,
+      appInfo.totalCoreSec,
```

Comment:
can we move this to be the last column in the CSV output, that way if someone is relying on the order it won't break them.

Reply:
fixed, thanks